packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -1,10 +1,78 @@
 /**
- * @typedef AppealType
+ * @typedef LPAAppealData
```

Comment:
Good to have this defined! Hoping to remove the need for these once we've got types defined by the API properly that we can reference.